packages/demo-site/index.ejs

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf8" />
+    <meta name="viewport" content="width=device-width" />
+
+    <title>Web Components</title>
+    <style>
+      #search-widget {
+        max-width: 1024px;
+        margin: 0 auto;
+      }
+    </style>
+    <link rel="stylesheet" href="<%= CDN_URL %>/search-widget.css" />
+  </head>
+
+  <body>
+    <div id="search-widget"></div>
+    <script src="<%= CDN_URL %>/search-widget.js"></script>
```

Comment:
There is a theme js file that powers the variants, would be great if we can include it too see: https://github.com/reapit/foundations/pull/785/files#diff-f2a26bcb9bf74e754adffd0909c8e28aR1